app/src/main/java/com/mapbox/maps/testapp/examples/CircleLayerClusteringActivity.kt

```diff
@@ -34,35 +34,35 @@ class CircleLayerClusteringActivity : AppCompatActivity() {
     val mapboxMap = mapView.getMapboxMap()
 
     mapboxMap.loadStyleUri(
-      Style.LIGHT
-    ) {
+      styleUri = Style.LIGHT,
       // Disable any type of fading transition when icons collide on the map. This enhances the visual
       // look of the data clustering together and breaking apart.
-      it.styleTransition = TransitionOptions.Builder()
+      styleTransitionOptions = TransitionOptions.Builder()
         .duration(0)
         .delay(0)
         .enablePlacementTransitions(false)
-        .build()
+        .build(),
+      onStyleLoaded = {
```

Comment:
nit: can it be an ending block without the `onStyleLoaded =`?

Reply:
sadly it can't as in that case lambda `{}` is ambiguous as we have 2 callbacks so that's the only way.